src/Sulu/Bundle/HttpCacheBundle/DependencyInjection/SuluHttpCacheExtension.php

```diff
@@ -14,17 +14,31 @@
 use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 
-class SuluHttpCacheExtension extends Extension
+/**
+ * Container extension for sulu-http-cache bundle.
+ */
+class SuluHttpCacheExtension extends Extension implements PrependExtensionInterface
 {
     /**
      * {@inheritdoc}
      */
-    public function getConfiguration(array $config, ContainerBuilder $container)
+    public function prepend(ContainerBuilder $container)
     {
-        return new Configuration();
+        if ('prod' === $container->getParameter('kernel.environment')) {
+            $container->prependExtensionConfig(
+                'sulu_http_cache',
+                [
+                    'handlers' => [
+                        'public' => ['enabled' => false],
+                        'url' => ['enabled' => false],
```

Comment:
Ah, I see, it is disabled by default... Could imagine that we change that once we upgrade to Symfony 3.2, since the cache will support that as well then. So I would also list it here.